ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -224,6 +224,19 @@ const PanelMain = fp.flow(
     const [analysisConfig, setAnalysisConfig] = useState(
       withAnalysisConfigDefaults(existingAnalysisConfig, gcePersistentDisk)
```

Comment:
maybe setting `useState()` with values which already have complex state interactions is the problem here